src/parser.cpp

```diff
@@ -479,6 +479,103 @@ options_metadata parser::load_settings()
         "node.refresh_transactions",
         value<bool>(&configured.node.refresh_transactions),
         "Request transactions on each channel start, defaults to false."
+    )
+
+    /* [bitcoin] */
+    (
+        "bitcoin.retargeting_factor",
+        value<uint32_t>(&configured.bitcoin.retargeting_factor),
+        "The retargeting factor, defaults to 4."
```

Comment:
"The difficulty retargeting factor, defaults to 4."

Reply:
Done.